### ISSUE TITLE
Add institution icon to menu bar at top of each window

### ISSUE THREAD

**Issue by randystern** [ProjectMirador/mirador#376]
As a researcher, I need to be able to know which version of a document I am looking at in a given window so that i don't get confused about which window is which document. The institution icon will help in that regard.

**Comment by regisrob** [comment]
somehow related to that: be able to see the complete label of a given object (e.g. shelfmark) without having to clickthe metadata icon, maybe by showing a tooltip while hovering the window title bar. Especially useful for freeform view

**Comment by regisrob** [comment]
If present in the manifest, the logo of the institution could be appended to the manifest title in the "manifest-info" bar (between the icon(s) and the manifest title)

**Comment by regisrob** [comment]
...but there is no way to know whether the image given as "logo" is actually the logo of the institution... or if the image is 2000x2000 or its weight 10 Mo... :grimacing: (even if the spec recommends a "small image" available via a IIIF Image API service).
On the other hand, having the link to the logo in the metadata panel is not very useful for the users...
Branding may be better handled by each implementer into their customized M2 instance.
Thoughts ?

**Comment by randystern** [comment]
I had been thinking of the same image that is displayed in the select view, but requested through the IIIF image API scaled to be height of the label/menu bar and displayed to the left of the label at the top of the viewing panel..

**Comment by regisrob** [comment]
I agree, it would be the ideal case. I made a quick test with a static image (not delivrered via IIIF, and scaled with css to fit the height of the title bar) in this demo: http://iiif.biblissima.fr/manifests/view/ark:/12148/btv1b55002481n (temporary instance of M2)

**Comment by randystern** [comment]
Yes!

**Comment by mejackreed** [comment]
I'm closing this issue for now. In Mirador 3 we originally did this in #1809. However, we ended up removing this feature. What we found, with just a few IIIF institutions that logos provided were so inconsistent, providing a good user experience was almost impossible. Removed in #1995.

Screenshots in both PR's for context. We do render the logo in a sidepanel, that can be configured to be displayed by default.
